utils/globalvars.go

```diff
@@ -59,6 +60,21 @@ func LoadEnvironmentVariables(ctx context.Context) (err error) {
 			CleanUpRoutineInterval = dur
 		}
 	}
+	ConcurrencyWorkers, _ = ParseIntEnvVar(ConcurrencyEnvironmentVariable, ConcurrencyWorkers)
 
 	return nil
 }
+
+func ParseIntEnvVar(varName string, defaultValue int) (int, error) {
```

Comment:
This function was [copied as-is from the KS repo](https://github.com/kubescape/kubescape/blob/cf08daf7fbb17c799325927f9f0898df05aa4b22/core/cautils/strutils.go#L53).
Since we don't have a global pkg for such functions, I simply inserted it here.
I will change the function to private and add unit tests.